packages/storybook/cxl-ui/cxl-section-hp-hero.stories.js

```diff
@@ -0,0 +1,49 @@
+import { html } from 'lit';
+import '@conversionxl/cxl-ui/src/components/cxl-section.js';
+import '@vaadin/button';
+import '@vaadin/icon';
+import '@vaadin/horizontal-layout';
+
+const meta = {
+  component: 'cxl-section',
+  title: 'CXL UI/cxl-section-hp-hero',
+};
+
+export default meta;
+
+export const CXLSectionHPHero = {
+  args: {
+    backgroundColor: 'var(--lumo-shade)',
+    hasWave: false,
+  },
+  name: 'cxl-section',
+  render: ({ backgroundColor, hasWave, content }) => html`
+    <cxl-section
+      class="alignwide has-background ${hasWave ? 'has-wave' : ''}"
+      style="${backgroundColor ? `background-color: ${backgroundColor}` : ''}"
+    >
+      <div class="is-layout-flex wp-container-3 wp-block-columns">
+        <div class="is-layout-flow wp-block-column">
+          <h1 class="entry-title has-white-color has-text-color" id="h-top-1-marketersteach-you-what-they-know"><strong>Top 1% marketers</strong> teach you what they know.</h1>
+          <h3 class="has-white-color has-text-color" id="h-hand-picked-instructors-peer-vetted-forexpertise-and-success-record">Hand picked instructors peer vetted for expertise and success record.</h3>
+          <p class="has-white-color has-text-color">Expert online marketing courses on customer acquisition, brand &amp; product marketing, conversion optimization, and digital analytics.</p>
+          <p>
+            <vaadin-button theme="primary large">
+              <vaadin-icon slot="prefix" icon="lumo:angle-right"></vaadin-icon>Start 7-day trial for $1
+            </vaadin-button>
+          </p>
+          <p class="has-white-color has-text-color has-small-font-size">Starting from $89/month.</p>
+        </div>
+        <div class="is-layout-flow wp-block-column">
+          <div class="wp-block-image">
+            <figure class="alignleft size-large">
+              <img decoding="async" src="https://cxl.com/institute/wp-content/uploads/2023/03/Carlos_Hero_Picture_BW-_kjid_Circle_01.png" alt="" data-src="https://cxl.com/institute/wp-content/uploads/2023/03/Carlos_Hero_Picture_BW-_kjid_Circle_01.png" class="lazyloaded"><noscript><img decoding="async" src="https://cxl.com/institute/wp-content/uploads/2023/03/Carlos_Hero_Picture_BW-_kjid_Circle_01.png" alt="" data-eio="l" /></noscript>
+            </figure>
+          </div>
+          <p class="has-white-color has-text-color has-small-font-size">“CXL influenced a quantum leap in my career. After 10 months of study, I was hired alongside some of the most brilliant minds in experimentation to work with leading brands from all over the world.”</p>
+          <p class="has-white-color has-text-color has-small-font-size"><strong>Carlos Trujillo</strong>,<br>Experimentation<br>Strategist @ Speero</p>
+        </div>
+      </div>
```

Comment:
Column system is the primary remaining challenge?
Can you post a TODO checklist in PR opener as you see it?

Quick experiment with loading BE CSS rendered quite well

```diff
diff --git a/packages/storybook/.storybook/preview-head.html b/packages/storybook/.storybook/preview-head.html
index 699736e5..3e581715 100644
--- a/packages/storybook/.storybook/preview-head.html
+++ b/packages/storybook/.storybook/preview-head.html
@@ -4,3 +4,4 @@
   delete Document.prototype.adoptedStyleSheets;
 </script>
 <link rel="stylesheet" id="cxl-font-css"  href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i,900|Libre+Baskerville:400,400i,700" type="text/css" media="all">
+<link rel="stylesheet" id="wp-block-library-css" href="https://cxl.com/institute/wp-includes/css/dist/block-library/style.min.css?ver=6.1.1" type="text/css" media="all">
```

(Requires SB full restart.)

:exclamation: I looked at unminified https://cxl.com/institute/wp-includes/css/dist/block-library/style.css?ver=6.1.1
It has a metric ton of crap. I think we need to just take frontend block styling over in our Storybook, as we don't really use that many blocks.

:exclamation: Beyond general BE styling, specifically for Columns, we still have Pure CSS vs Block Editor. Among other things, breakpoints are different.

:heavy_check_mark: I will attempt a mu-plugin in the spirit of cxl-section. Or it might be these integrations should migrate into wp-theme-lib CXL UI integration, that actually makes more sense.

Anything else you need to land this front page project?